frontend/common/src/components/Pending/Loading.tsx

```diff
@@ -38,9 +36,10 @@ const Loading = ({ inline = false, live }: LoadingProps): JSX.Element => {
       {...(live && {
         "aria-live": live,
       })}
-      data-h2-display="b(flex)"
-      data-h2-align-items="b(center)"
-      data-h2-justify-content="b(center)"
+      data-h2-background-color="base(dt-white.2)"
```

Comment:
Inline spinner should not include a background colour.

<img width="79" alt="Screen Shot 2022-08-18 at 10 47 35 AM" src="https://user-images.githubusercontent.com/4127998/185445425-72729d42-7acf-4d83-a87a-aee683284320.png">